mobile/lib/theme/dynamic_theme.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:flutter/material.dart';
+import 'package:dynamic_color/dynamic_color.dart';
+
+import 'package:immich_mobile/theme/theme_data.dart';
+
+class DynamicTheme {
```

Comment:
This could very well be an abstract final class as all the members / fields are static

Reply:
Thanks for your reivew. I have made the change.